collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -576,6 +575,8 @@ public void setTaskListener(TaskListener tl) {
             String requestString = "'" + currentDir + "' in parents and trashed=false";
             Files.List request = null;
 
+            //// TODO: 19/3/17 update library
+/*
```

Comment:
@nribeka I asked @shobhitagarwal1612 to submit just this part of the change to take a quick look/sanity check. The rest is in #722 so let's go review there! I'm looking at it as well.